pkg/setup/setup.go

```diff
@@ -41,20 +41,20 @@ func SetupTest(t *testing.T, client *kubernetes.Clientset, tektonYAMLPath string
 	t.Logf("using namespace: %s", namespace)
 
 	// Cleanup function
-	cleanup := func() {
+	t.Cleanup(func() {
 		t.Helper()
```

Comment:
This doesn't need to be a helper I think.

Reply:
ah I see.